### DISCUSSION TITLE
Call for testing Freeplane preview 1.11.1_21

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/freeplane#1055] [Announcements]
Please check it.
If there are no show-stoppers I am going to release it as stable.

I have changed how the chosen layout is shown, I hope you like it.
I have also modified the navigation logic a bit.

Dimitry

**Comment by sifran-github** [comment]
Posted 3 bugs:
https://github.com/freeplane/freeplane/issues/1056 - Pressing ctrl+z shows incorrect layout icons
https://github.com/freeplane/freeplane/issues/1057 - Jump in & Jump out on folded nodes changes the map position
https://github.com/freeplane/freeplane/discussions/989#discussioncomment-5428775 - Pressing Up on outline layouts that goes up skips a node.

Edit: Another bug:
https://github.com/freeplane/freeplane/issues/1059 - Jumping in & Out causes Freeplane to crash

**Reply by sifran-github** [reply]
in freeplane-1.11.1-pre22 i think that these issues got fixed,
but not https://github.com/freeplane/freeplane/issues/1059 which is still open and crashes in the same way as before.

**Reply by sifran-github** [reply]
another small suggestion: like with every combobox, once the user selected (clicked) on a layout in the layout combobox, the combobox should be closed automatically.

**Reply by sifran-github** [reply]
https://github.com/freeplane/freeplane/issues/1057 is also not completely fixed. see my comments inside

**Reply by sifran-github** [reply]
as far as i can tell, freeplane-1.11.1-pre23 solves all of the issues mentioned here, but I will test a bit more.

**Reply by sifran-github** [reply]
opened https://github.com/freeplane/freeplane/issues/1061
might be related to the fix of https://github.com/freeplane/freeplane/issues/1057 but i'm not sure.

**Reply by sifran-github** [reply]
tested freeplane-1.11.1-pre24, all the issues mentioned above seem to be fixed.

**Reply by sifran-github** [reply]
opened https://github.com/freeplane/freeplane/issues/1064

**Comment by abc16361** [comment]
One of my maps uses script filter conditional styles and runs smoothly on fp-1.11.1-pre14, but on versions from pre16 to pre22, editing like typing and adding nodes would cause large memory consumption (~40% cpu). If the script filter conditional styles are disable, large memory consumption would not happen. On fp-1.11.1-pre14, editing just triggers a brief increase of memory concuption up to 20% and quickly dropping back to zero.

**Reply by dpolivaev** [reply]
Submit a bug report and attach a map that can allow me to reproduce the issue.

The problem might be related to a change introduced in commit https://github.com/freeplane/freeplane/commit/e8dc4a67a8897c7730e7f7d29db55215d96135f5 
to fix https://github.com/freeplane/freeplane/issues/1022

**Reply by dpolivaev** [reply]
By the way, do you mean memory consumption or CPU load?

**Reply by abc16361** [reply]
Just summited a [report](https://github.com/freeplane/freeplane/issues/1060#issue-1640999895).

**Reply by abc16361** [reply]
>By the way, do you mean memory consumption or CPU load?

It's the cpu usage on Windows Task Manager.

**Comment by Illegit** [comment]
I want to feedback that compared with 1.11.1_21 and previous ones, the actual 1.11.1_23 is much smoother and fluider. even with my complicate maps, very nice.
